neuron/src/lib/Neuron/Zettelkasten/Zettel/Parser.hs

```diff
@@ -37,14 +38,14 @@ parseZettel format zreader fn zid s = do
               ((,True) . plainify . snd <$> getH1 doc)
                 <|> ((,False) . takeInitial . plainify <$> getFirstParagraphText doc)
           tags = fromMaybe [] $ Meta.tags =<< meta
-          day = case zid of
+          date = case zid of
             -- We ignore the "data" meta field on legacy Date IDs, which encode the
             -- creation date in the ID.
-            ZettelDateID v _ -> Just v
+            ZettelDateID v _ -> Just (Right $ LocalTime v midday)
```

Comment:
Why are we automatically guessing a time (`midday`) here instead of using the `Left` day?